src/styles/blocks/card.scss

```diff
@@ -0,0 +1,93 @@
+.card {
+  box-sizing: border-box;
+  border: 1px solid #e5e5e5;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
```

Comment:
either change it to max-width, or use mix for that

Reply:
done